### PR TITLE
Add Codestyle configuration to Eclipse + Java code formatter

### PR DESCRIPTION
**Pull request by SupunArunoda**
Hi,
Formatter works when save/compile the code using eclipse.

### CODE REVIEW

File: org.eclipse.cpacep.ui/src/org/eclipse/internal/cpacep/view/CpacepView.java [hunk 1]
```diff
@@ -16,7 +17,8 @@
     private Text text;
 
     public void createPartControl(Composite parent) {
-	text = new Text(parent, SWT.READ_ONLY | SWT.MULTI);
```
Comment (leostrakosch) [inline org.eclipse.cpacep.ui/src/org/eclipse/internal/cpacep/view/CpacepView.java]: This does not belong in this pull request

File: org.eclipse.cpacep.ui/src/org/eclipse/internal/cpacep/view/CpacepView.java [hunk 2]
```diff
@@ -16,7 +17,8 @@
...
+	text = new Text(parent, SWT.H_SCROLL | SWT.V_SCROLL | SWT.READ_ONLY);
+
```
Comment (leostrakosch) [inline org.eclipse.cpacep.ui/src/org/eclipse/internal/cpacep/view/CpacepView.java]: This does not belong in this pull request

**Comment by leostrakosch** [comment]
Nice, could you also write a maven goal that executes the google java formatter? Not everyone is using eclipse.

**Comment by leostrakosch** [comment]
Oh, and we should still adjust the code style in eclipse itself! People may not know about the google formatter and thus not use it. Then, we still want them to adhere to a certain level of code quality.

You can probably reuse much of the settings used in [sosy-lab/java-commonlib](https://github.com/sosy-lab/java-common-lib):

https://github.com/sosy-lab/java-common-lib/tree/6a2bf0721986fb112db8193f5efbb67ce1b392b0/.settings

**Comment by SupunArunoda** [comment]
Hi Thomas,
We don't need to add formatter xml to eclipse. Maven plugin will take care of them

**Comment by leostrakosch** [comment]
Alright, in Eclipse it works really well! Is there a way to deactivate this, if I want to force some specific formatting for some reason? (if not, I'd still be happy with it).

We still need a way to run the formatter as a maven goal, in case people don't use Eclipse, though.

**Comment by SupunArunoda** [comment]
Hi Thomas,
Added code formatter goal. Execute **mvn formatter:format** when need to do the formatting.

**Comment by leostrakosch** [comment]
Nice, looks good! 👍 
Should we do a full reformat after the open PRs are merged (only after to avoid merge conflicts)?
